earth_enterprise/src/installer/install_server.sh

```diff
@@ -53,6 +45,20 @@ PRODUCT_NAME="$GEE"
 START_SERVER_DAEMON=1
 PROMPT_FOR_START="n"
 
+# user names
+DEFAULTGROUPNAME="gegroup"
+GRPNAME=$DEFAULTGROUPNAME
+GEAPACHEUSER_NAME="geapacheuser"
+GEPGUSER_NAME="gepguser"
+DEFAULTGEFUSIONUSER_NAME="gefusionuser"
```

Comment:
These two lines don't need to be in server install as fusion is not required for server